apps/entropy-tester/src/index.ts

```diff
@@ -0,0 +1,194 @@
+import fs from "node:fs/promises";
+
+import type { PrivateKey } from "@pythnetwork/contract-manager/core/base";
+import { toPrivateKey } from "@pythnetwork/contract-manager/core/base";
+import { EvmEntropyContract } from "@pythnetwork/contract-manager/core/contracts/evm";
+import { DefaultStore } from "@pythnetwork/contract-manager/node/store";
+import type { Logger } from "pino";
+import { pino } from "pino";
+import type { ArgumentsCamelCase, InferredOptionTypes } from "yargs";
+import yargs from "yargs";
+import { hideBin } from "yargs/helpers";
+
+type LoadedConfig = {
+  contract: EvmEntropyContract;
+  interval: number;
+};
+
+function timeToSeconds(timeStr: string): number {
+  const match = /^(\d+)([hms])$/i.exec(timeStr);
+  if (!match?.[1] || !match[2])
+    throw new Error("Invalid format. Use formats like '6h', '15m', or '30s'.");
+
+  const value = Number.parseInt(match[1], 10);
+  const unit = match[2].toLowerCase();
+
+  switch (unit) {
+    case "h": {
+      return value * 3600;
+    }
+    case "m": {
+      return value * 60;
+    }
+    case "s": {
+      return value;
+    }
+    default: {
+      throw new Error("Unsupported time unit.");
+    }
+  }
+}
+
+type ParsedConfig = {
+  "chain-id": string;
+  interval: string;
+};
+
+async function loadConfig(configPath: string): Promise<LoadedConfig[]> {
+  const configs = (await import(configPath, {
+    with: { type: "json" },
+  })) as { default: ParsedConfig[] };
```

Comment:
For all the same reasons you use serde to deserialize in Rust and you don't just unsafe cast external data, we should also avoid typecasting in typescript and should instead validate the schema using zod: https://v3.zod.dev/ (note we still use v3 everywhere in the monorepo but you're welcome to try v4 if you want).  This would look basically like this:

```ts
const configSchema = z.array(z.strictObject({
  "chain-id": z.string(),
  interval: z.string(),
}));

// ...

const configs = configSchema.parse(
  (await import(configPath, { with: { type: 'json' } })).default
);
```